sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -613,8 +614,15 @@ public AnalyzedRelation visitTableFunction(TableFunction node, StatementAnalysis
         }
         Function function = (Function) symbol;
         FunctionIdent ident = function.info().ident();
+        Signature signature = function.signature();
 
-        FunctionImplementation functionImplementation = functions.getQualified(ident);
+        FunctionImplementation functionImplementation;
+        if (signature == null) {
+            functionImplementation = functions.getQualified(ident);
```

Comment:
This case will be removed once all functions are migrated,  right?

Reply:
Yep